yucca/functional/evaluation/evaluate_folder.py

```diff
@@ -0,0 +1,281 @@
+import sys
+import numpy as np
+import nibabel as nib
+import logging
+from typing import Optional
+from yucca.functional.transforms.label_transforms import convert_labels_to_regions, translate_region_labels
+from yucca.functional.utils.nib_utils import get_nib_spacing
+from yucca.functional.evaluation.obj_metrics import get_obj_stats_for_label
+from yucca.functional.evaluation.surface_metrics import get_surface_metrics_for_label
+from tqdm import tqdm
+from batchgenerators.utilities.file_and_folder_operations import join
+from sklearn.metrics import confusion_matrix
+from yucca.functional.evaluation.metrics import auroc
+
+
+def evaluate_multilabel_folder_segm(
```

Comment:
I think this is so cool to have. I think it would be nice to spend just a few minutes rewriting this code, so it is a bit less duplicated and I think those functions might also be useful in Agave and other projects.